packages/pharos/src/components/input-group/pharos-input-group.scss

```diff
@@ -37,3 +37,21 @@ slot[name='prepend']::slotted([data-pharos-component='PharosIcon']:first-child)
 slot:not([name])::slotted([data-pharos-component='PharosIcon']:last-child) {
   margin-right: calc(var(--pharos-spacing-three-quarters-x) + 1px);
 }
+
+:host([variant='cozy']) {
+  slot[name='prepend']::slotted(*) {
+    margin-left: var(--pharos-spacing-1-x);
+  }
+
+  slot[name='prepend']::slotted([data-pharos-component='PharosIcon']:first-child) {
+    margin-left: var(--pharos-spacing-1-x);
+  }
+
+  slot:not([name])::slotted(:last-child) {
+    margin-right: var(--pharos-spacing-1-x);
+  }
+
+  slot:not([name])::slotted([data-pharos-component='PharosIcon']:last-child) {
+    margin-right: var(--pharos-spacing-1-x);
+  }
+}
```

Comment:
I think these should all be `--pharos-spacing-three-quarters-x`

Reply:
i thought based on the conversion we want to be uniformally 1-x everywhere?

Reply:
updated